plugins/ocm-common/README.md

```diff
@@ -1,5 +1,5 @@
 # Open Cluster Management plugin for Backstage
 
-This plugin integrates your Backstage instance with Open Cluster Management.
+The Open Cluster Management (OCM) plugin integrates your Backstage instance with Open Cluster Management.
 
-This is just a part of the plugin, for documentation please refer to [`@janus-idp/backstage-plugin-ocm` documentation](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/ocm-backend).
+For OCM plugin documentation, see [Open Cluster Management plugin documentation](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/ocm) on GitHub.
```

Comment:
same comment as previous since this content is the same - might be worth looking into extensions here to introduce snippets and prevent content duplication?